docker-compose.yml

```diff
@@ -21,7 +20,17 @@ services:
       - APP_RESULT_DIR=/data/
     restart: always
     volumes:
-      - ./outputdata:/data
+      - ./redisdata:/data
+
+#  cubequery-worker:

```

Comment:
wouldn't be better to remove these lines rather than comment them out?

Reply:
Probably shouldn't be commented out, I disabled the worker as it was faster to test...